components/layout/index.js

```diff
@@ -13,15 +13,17 @@ const Layout = ({ children, title, description }) => {
 
   return (
     <div>
-      <Head>
-        <title>{title}</title>
-        <meta name="description" content={description} />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-      </Head>
-      <Navbar drawerToggleClicked={sideDrawerToggleHandler} />
-      <SideDrawer open={showSideDrawer} close={sideDrawerToggleHandler} />
-      <div className="container">
-        <div>{children}</div>
+      <div>
```

Comment:
I don't see any changes in this file, is this indentation changes

Reply:
I had created a new scss file in an earlier commit and so to revert back to normal changes without the creation of any component, I set the code to what it was initially. I removed these three lines of code and reset it to what it was at the very beginning.

![ss](https://user-images.githubusercontent.com/50043126/128456378-6112ff4a-6164-4ec2-a1e6-8309dcc47db8.png)

Reply:
I think the PR you are reviewing is an old one. Please have a look at the latest PR for better understanding.

Reply:
> I don't see any changes in this file, is this indentation changes

Looks like when I was reverting back to the original code of index.js (that is, to what it was originally), I forgot to remove the extra **div**  tags that I had created. I will apply the changes